src/v/kafka/server/group_manager.cc

```diff
@@ -152,6 +152,8 @@ void group_manager::handle_topic_delta(
           [this](const std::vector<model::topic_partition>& tps) {
               return cleanup_removed_topic_partitions(tps);
           });
+    }).handle_exception([](std::exception_ptr e) {
+        vlog(klog.info, "Topic clean-up encountered error: {}", e);
```

Comment:
Can you describe when this can occur, and the implications? Is `info` the right level?

Reply:
It occurs anytime `cleanup_removed_topic_partitions (e.g. io error)` returns an exceptional future. But since the whole thing runs in the background, this is the only way to know something went wrong.

Good call on `info` -- should be warning.